templates/publisher/measure.html

```diff
@@ -73,14 +77,18 @@ <h3 class="u-float--left">Daily active devices</h3>
         <div class="col-6">
           <div class="u-clearfix">
             <h3 class="u-float--left">Territories</h3>
-            <strong class="p-heading--three u-float--right u-no-margin--top">74</strong>
+            <div class="p-heading--three u-float--right u-no-margin--top">
```

Comment:
@Lukewh I don't see this when I reload the demo, so might be wrong, but when I move the p-heading-three class onto the div, the max width limits it and the number moves. What I meant was, wrap the strong in a <p> -
 <p class="p-heading--three"><strong>{{number}}</strong></p>

![image](https://user-images.githubusercontent.com/2741678/34561283-2f5605fe-f141-11e7-8b18-5ab994ed51d3.png)

Please ignore this if you've found another way to get it to look correctly.

Reply:
I can't reproduce this @lyubomir-popov I'm going to merge and if we notice it while I'm working on the other graphs we can revisit!